### PR TITLE
igmspec_class

### PR DESCRIPTION
**Pull request by profxj**
As titled.

This is to package the functionality of igmspec into a single
simple Class

### CODE REVIEW

File: igmspec/igmspec.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        # Init
+        self.qcat = QueryCatalog(**kwargs)
+        self.idb = InterfaceDB(**kwargs)
+        # Checks
+        assert self.idb.db_file == self.qcat.db_file
+        for survey in self.idb.surveys:
+            assert survey in self.qcat.surveys
+
+    def spec_from_coord(self, coord, toler=5.*u.arcsec, isurvey=None, **kwargs):
+        """ Radial search for spectra around given coordinate
```
Comment (ntejos) [inline igmspec/igmspec.py]: Can you use `tol` rather than `toler` for conforming the linetools convention?

File: igmspec/interface_db.py [hunk 2]
```diff
@@ -165,27 +165,58 @@ def grab_spec(self, survey, IGM_IDs, verbose=None, **kwargs):
...
         spec = XSpectrum1D(data['wave'], data['flux'], sig=data['sig'], masking='edges')
         # Return
-        return spec
+        return spec, self.meta
+
+    def show_meta(self, imeta=None, meta_keys=None):
+        """ Nicely format and show the meta table
+        Parameters
+        ----------
+        meta_keys : list, optional
```
Comment (ntejos) [inline igmspec/interface_db.py]: Please specify the `imeta` in the docstring.

**Comment by ntejos** [comment]
Things look good to me, apart from improving the docstring in a a couple of cases.

File: igmspec/igmspec.py (hunk 1 above)
Comment (profxj) [inline igmspec/igmspec.py]: Yup

File: igmspec/interface_db.py (hunk 2 above)
Comment (profxj) [inline igmspec/interface_db.py]: Done.

**Comment by profxj** [comment]
Merging

**Pull request merged**